packages/framework/src/Support/Filesystem/MediaFile.php

```diff
@@ -97,27 +97,21 @@ public function toArray(): array
 
     public function getContentLength(): int
     {
-        if (! isset($this->length)) {
-            $this->boot();
-        }
+        $this->ensureInstanceIsBooted('length');
```

Comment:
Technically we could do something like `$this->getBootedProperty()` but I think this is much more readable and understandable.